client/post-editor/post-editor.jsx

```diff
@@ -231,9 +240,15 @@ export const PostEditor = React.createClass( {
 			<div className={ classes }>
 				<QueryPreferences />
 				<EditorConfirmationSidebar
-					hideSidebar={ this.hideConfirmationSidebar }
-					isActive={ this.state.showConfirmationSidebar }
+					closeOverlay={ this.closeConfirmationSidebarOverlay }
+					closeSidebar={ this.closeConfirmationSidebar }
+					isOverlayActive={ this.state.showConfirmationSidebarOverlay }
+					isSidebarActive={ this.state.showConfirmationSidebar }
```

Comment:
This change was forward-looking for the first task in 1a003-pb, where the sidebar closes but the overlay remains while we leave behind a "PUBLISHING..." busy-button.